### PR TITLE
Added album covers to the musicbrainz api

### PR DESCRIPTION
**Pull request by ltctceplrm**
To complete issue #113.

I also added the missing manga media type since I couldn't get the plugin to build otherwise.

### CODE REVIEW

**Comment by ltctceplrm** [comment]
I intended to create a new pull request to amend #106 but it seems I added it to this commit instead, I hope that's still okay.

**Comment by mProjectsCode** [comment]
Thanks for the PR.
Ping me once you are done so that i can merge this. :)

**Comment by ltctceplrm** [comment]
You're welcome, I was thinking of doing #89 as well, but I have some questions about that:
Both writer and director are included in OMDbapi adding the writer would be easy but the director is currently already placed under the producer field in the plugin, would it be okay if I renamed `producer` to `director` in the movies model and the OMDbapi? Director would be more correct but unfortunately MALapi doesn't give director and writer information so I'd have to add an empty string for movies grabbed with MALapi.

Would this be acceptable or should I just add writer and leave the director as `producer`?

An extra question is if I do change the producer field in movies, should it also be renamed in series? I can add writer and director fields for TV series as well and for anime grabbed by MALapi I can leave them empty again + rename producer to studio since it currently grabs the studio info and puts it under the producer field.

**Comment by mProjectsCode** [comment]
I would say yes, as it seems to be requested by many.
And adding the same changes to series seems like a good idea.

**Comment by ltctceplrm** [comment]
There, it should be done now @mProjectsCode.

File: src/models/BookModel.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { MediaType } from '../utils/MediaType';
 
 export class BookModel extends MediaTypeModel {
 	author: string;
+	plot:string;
```
Comment (mProjectsCode) [inline src/models/BookModel.ts:7]: small formatting mistake, running `npm run format` should fix this

Comment (ltctceplrm) [reply src/models/BookModel.ts]: Ah good to know, I ran it and it also fixed a line end in MovieModel.ts

**Comment by mProjectsCode** [comment]
LGMT, will merge now and go over the code and maybe release it tomorrow

**Comment by ltctceplrm** [comment]
Awesome, thank you 😄

**Pull request merged**